internal/alert/provider/discord/send.go

```diff
@@ -0,0 +1,22 @@
+package discord
+
+import (
+	"github.com/balerter/balerter/internal/alert/message"
+	"github.com/bwmarrin/discordgo"
+)
+
+// Send implements
+func (d *Discord) Send(mes *message.Message) error {
+
+	c, err := discordgo.New("Bot " + d.conf.Token)
```

Comment:
Maybe, we should create a client in the 'New' func, store it in the Discord sctruct and use in Send func?

Reply:
Yip I thought about that too but wasn't sure how to close the session at the end. Mby just let it auto expire? Will do this.